packages/zendesk-adapter/src/filters/guide_arrange_paths.ts

```diff
@@ -79,6 +91,62 @@ export const GUIDE_ELEMENT_DIRECTORY: Record<string, string> = {
   [GUIDE_LOCALE]: 'locale',
 }
 
+
+const getTranslationLocale = (instance?: InstanceElement): string => {
+  if (instance === undefined) {
+    return NO_VALUE_DEFAULT
+  }
+  if (isReferenceExpression(instance.value.locale)) {
+    return instance.value.locale.value.value?.id ?? NO_VALUE_DEFAULT // will have to change when locale is changed
+  }
+  return instance.value.locale ?? NO_VALUE_DEFAULT
+}
+
+const getNameFromTranslation = (instance?: InstanceElement): string => {
+  if (instance === undefined) {
+    return NO_VALUE_DEFAULT
+  }
+  const sourceLocale = isReferenceExpression(instance.value.source_locale)
+    ? instance.value.source_locale.value.value?.id
+    : instance.value.source_locale
+  const translation = instance.value.translations
+    ?.filter(isReferenceExpression)
+    .map((reference: ReferenceExpression) => reference.value)
+    .find((tran: InstanceElement) => (isReferenceExpression(tran.value.locale)
+      ? tran.value.locale.value.value?.id === sourceLocale
+      : tran.value.locale === sourceLocale))
+  return translation?.value.title ?? NO_VALUE_DEFAULT
+}
+
+const getNameFromTitle = (instance?: InstanceElement): string => {
+  if (instance === undefined) {
+    return NO_VALUE_DEFAULT
+  }
+  return instance.value.name ?? NO_VALUE_DEFAULT
```

Comment:
nit: I think this can be combined
```suggestion
  return instance?.value.name ?? NO_VALUE_DEFAULT
```